*** ./smali/android/preference/PreferenceActivity.smali	2014-08-24 16:08:52.213929769 -0500
--- /home/albert/patchrom/jewel/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceActivity.smali	2014-08-24 16:13:05.225377512 -0500
***************
*** 2147,2152 ****
--- 2147,2154 ----
  
      if-lez v17, :cond_e
  
+     invoke-static/range {p0 .. p0}, Landroid/preference/Injector$PreferenceActivityHook;->setListViewPadding(Landroid/preference/PreferenceActivity;)V
+ 
      new-instance v17, Landroid/preference/PreferenceActivity$HeaderAdapter;
  
      move-object/from16 v0, p0
