*** ./smali/com/android/internal/telephony/cdma/CdmaServiceStateTracker.smali	2014-08-24 16:05:50.735442859 -0500
--- /home/albert/patchrom/jewel/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/cdma/CdmaServiceStateTracker.smali	2014-08-24 16:09:53.382765341 -0500
***************
*** 8523,8528 ****
--- 8523,8532 ----
      move-result-object v1
  
      .local v1, plmn:Ljava/lang/String;
+     invoke-static {p0, v1}, Lcom/android/internal/telephony/Injector$ServiceStateTrackerHook;->getSpn(Lcom/android/internal/telephony/ServiceStateTracker;Ljava/lang/String;)Ljava/lang/String;
+ 
+     move-result-object v1
+ 
      iget-object v5, p0, Lcom/android/internal/telephony/cdma/CdmaServiceStateTracker;->mCurPlmn:Ljava/lang/String;
  
      invoke-static {v1, v5}, Landroid/text/TextUtils;->equals(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Z
