*** ./smali/android/app/INotificationManager.smali	2014-08-24 16:08:19.077476574 -0500
--- /home/albert/patchrom/jewel/temp/new_smali/framework.jar.out/./smali/android/app/INotificationManager.smali	2014-08-24 16:12:32.168928301 -0500
***************
*** 15,20 ****
--- 15,28 ----
  
  
  # virtual methods
+ .method public abstract areNotificationsEnabled(Ljava/lang/String;I)Z
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract areNotificationsEnabledForPackage(Ljava/lang/String;I)Z
      .annotation system Ldalvik/annotation/Throws;
          value = {
