*** ./smali/android/widget/Toast$TN.smali	2014-08-24 16:07:44.240999690 -0500
--- /home/albert/patchrom/jewel/temp/new_smali/framework.jar.out/./smali/android/widget/Toast$TN.smali	2014-08-24 16:11:57.252453414 -0500
***************
*** 368,373 ****
--- 368,385 ----
      return-void
  .end method
  
+ .method public setType(I)V
+     .locals 1
+     .parameter "type"
+ 
+     .prologue
+     iget-object v0, p0, Landroid/widget/Toast$TN;->mParams:Landroid/view/WindowManager$LayoutParams;
+ 
+     iput p1, v0, Landroid/view/WindowManager$LayoutParams;->type:I
+ 
+     return-void
+ .end method
+ 
  .method public show()V
      .locals 2
  
