*** ./smali/android/os/PowerManager.smali	2014-08-24 16:07:25.592744235 -0500
--- /home/albert/patchrom/jewel/temp/new_smali/framework.jar.out/./smali/android/os/PowerManager.smali	2014-08-24 16:11:38.868203208 -0500
***************
*** 14,19 ****
--- 14,21 ----
  # static fields
  .field public static final ACQUIRE_CAUSES_WAKEUP:I = 0x10000000
  
+ .field public static final BRIGHTNESS_DIM:I = 0x14
+ 
  .field public static final BRIGHTNESS_OFF:I = 0x0
  
  .field public static final BRIGHTNESS_ON:I = 0xff
