*** ./smali/com/android/internal/statusbar/IStatusBarService.smali	2014-08-24 16:06:05.467645190 -0500
--- /home/albert/patchrom/jewel/temp/new_smali/framework2.jar.out/./smali/com/android/internal/statusbar/IStatusBarService.smali	2014-08-24 16:10:09.574986303 -0500
***************
*** 179,184 ****
--- 179,192 ----
      .end annotation
  .end method
  
+ .method public abstract setStatus(ILandroid/os/IBinder;Ljava/lang/String;Landroid/os/Bundle;)V
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract setSystemUiVisibility(II)V
      .annotation system Ldalvik/annotation/Throws;
          value = {
