*** ./smali/android/graphics/drawable/NinePatchDrawable.smali	2014-08-24 16:07:34.072860417 -0500
--- /home/albert/patchrom/jewel/temp/new_smali/framework.jar.out/./smali/android/graphics/drawable/NinePatchDrawable.smali	2014-08-24 16:11:47.256317381 -0500
***************
*** 254,259 ****
--- 254,265 ----
  
      iput-object v4, p0, Landroid/graphics/drawable/NinePatchDrawable;->mOpticalInsets:Landroid/graphics/Insets;
  
+     iget-object v4, p0, Landroid/graphics/drawable/NinePatchDrawable;->mNinePatchState:Landroid/graphics/drawable/NinePatchDrawable$NinePatchState;
+ 
+     iget-object v4, v4, Landroid/graphics/drawable/NinePatchDrawable$NinePatchState;->mPadding:Landroid/graphics/Rect;
+ 
+     iput-object v4, p0, Landroid/graphics/drawable/NinePatchDrawable;->mPadding:Landroid/graphics/Rect;
+ 
      :goto_0
      return-void
  
